src/main/java/bossmonster/exception/GameEndException.java

```diff
@@ -0,0 +1,13 @@
+package bossmonster.exception;
+
+public class GameEndException extends RuntimeException{
+    private final boolean playerWin;
+    public GameEndException(String msg, boolean isPlayerWin) {
```

Comment:
```suggestion
public class GameEndException extends RuntimeException {
    private final boolean playerWin;
    
    public GameEndException(String msg, boolean isPlayerWin) {
```
[구글컨벤션](https://google.github.io/styleguide/javaguide.html)을 참고해서 코드를 짜시면 더 좋은 코드가 될 것 같아요!